plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixTypeMapping.java

```diff
@@ -430,12 +431,14 @@ private DataTypeTest dateTests(
                 valuesList -> valuesList == null ? null : valuesList.stream().map(elementType::toTrinoQueryResult).collect(toList()));
     }
 
-    public static DataType<LocalDate> phoenixDateDataType()
+    @DataProvider
+    public Object[][] testTimestampDataProvider()
```

Comment:
Should it be called `sessionZoneProvider` (similar to other existing test classes)?

Also Kathmandu and default JVM zone seem missing? Any specific reasons?